src/calendar/CalendarState.ts

```diff
@@ -29,13 +29,15 @@ import {
 import { CalendarProps } from "./index.d";
 import { useWeekStart } from "./useWeekStart";
 import { announce } from "../utils/LiveAnnouncer";
-import { generateDaysInMonthArray, isInvalid, useWeekDays } from "./__utils";
+import { isInvalid, useWeekDays, generateDaysInMonthArray } from "./__utils";
 
-export interface IUseCalendarProps extends CalendarProps {
+export type DateValue = string | number | Date;
```

Comment:
Seems like we are using DateValue repeatedly.
I can see this exported in index.d.ts & in Datepicker as well.

Can we create `utils/types.ts` and get it from a common place?